packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -180,11 +194,16 @@ export default class ReconnectionManager {
   media(meeting) {
     this.tryCount += 1;
     LoggerProxy.logger.log(`ReconnectionManager:index#media --> Begin reestablishment of media at ${new Date()}`);
+    let reCreatePeerConnection = false;
```

Comment:
Minor note. Recreate is one word so the variable would be `recreatePeerConnection`